partner_multi_relation/tests/__init__.py

```diff
@@ -0,0 +1,6 @@
+# Copyright 2016 Therp BV
```

Comment:
@ddufresne Remove Copyright from `init` file

Reply:
@nikul-serpentcs sorry, won't do that. I am not sure about OCA's policy about that and otherwise, I am not confortable removing someone's copyright from a python file.

Even though, I should not be the person commiting such change. Also, this should not block the PR.

Reply:
Well, it was a mistake to put copyright in a file that doesn't contain code, but it's not blocking either.

Reply:
We followed older guidelines to put copyright in ALL python files. I have no problem with removing it here and from any other __init__.py file.